client/reducers/index.js

```diff
@@ -66,6 +66,20 @@ export const getMessagesByStory = (state:Object, storyUid:string) => {
   return R.pickBy(isFromStory, getMessages(state))
 }
 
+export const byDateDescending = R.comparator((d1, d2) => d1.sent > d2.sent)
+export const getGroupMessages = (state, {params}) => {
+  const groupUid = params.groupUid;
+  return {
+    messages: R.sort(byDateDescending, R.values(getMessagesByGroup(state, groupUid))),
+  }
+}
+export const getStoryMessages = (state, {params}) => {
```

Comment:
I've changed this to `storyUid` / `groupUid` instead of `{params}` but it doesn't seem to be working.  Where does that function get called and how do I force it to send guid instead of an entire object?